src/AddImageChecks.cpp

```diff
@@ -387,8 +387,8 @@ Stmt add_image_checks(Stmt s,
                         << "as the first output buffer.\n";
 
                     stride_constrained = param.stride_constraint(i);
-                } else if (image.defined() && (int)i < image.dimensions()) {
-                    stride_constrained = image.dim(i).stride();
+                } else if (image.defined() && (int)i < image->dimensions()) {
```

Comment:
It's a little weird to have something called "SomethingRef" using the -> operator, but I guess I understand why...